packages/devextreme/testing/testcafe/tests/scheduler/layout/templates/appointmentTemplate.ts

```diff
@@ -66,3 +76,274 @@ fixture.disablePageReloads`Layout:Templates:appointmentTemplate`
     });
   });
 });
+
```

Comment:
Question.

Maybe we should create a separate fixture/file in the same folder for this matrix test?
Because it has a lot of initialization and variables in the module/file scope.